console/src/app/pages/projects/apps/app-create/app-create.component.scss

```diff
@@ -1,3 +1,32 @@
+@use '@angular/material' as mat;
+
+@mixin app-create-theme($theme) {
+  $primary: map-get($theme, primary);
+  $warn: map-get($theme, warn);
+  $background: map-get($theme, background);
+  $foreground: map-get($theme, foreground);
+  $accent: map-get($theme, accent);
+  $primary-color: mat.get-color-from-palette($primary, 500);
+
+  // Reference: https://github.com/angular/components/issues/10681#issuecomment-695185806
+  @for $i from 1 through 3 {
```

Comment:
lets let us wrap this in a class such that we only apply those styles to this stepper.. otherwise we should define a global style.. I'd add a `.app-create-wrapper` to the `cnsl-create-layout` 👍